include/upload/UploadServer.h

```diff
@@ -25,23 +28,29 @@ class UploadServer
 
     static void handleOptions(const http_request& request);
 
-    static faabric::Message buildMessageFromRequest(
-      const http_request& request);
-
-    static std::vector<std::string> getPathParts(const http_request& request);
-
-    static std::vector<uint8_t> getState(const http_request& request);
-
   private:
     bool stopped = false;
 
-    static void handlePythonFunctionUpload(const http_request& request);
+    static std::vector<uint8_t> getState(const std::string& user,
```

Comment:
Remember that `static` here means a `static` class variable (not `static` in the sense of global variables/ functions), so I'm not sure what relevance its `private` status has (i.e. you can have `public` static class methods too). The reason these methods are `static` is because the `cpprestsdk` requires that requests are handled by using static methods (e.g. [`handleGet`](https://github.com/faasm/faasm/blob/master/include/upload/UploadServer.h#L22)), and these methods are called from within those.